packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -101,7 +101,7 @@ exports.onPostBuild = (args, pluginOptions) => {
       },
       {
         // Use the Network First handler for external resources
-        urlPattern: /^https:/,
+        urlPattern: /^https?:/,
```

Comment:
Is this a best practice? This seems significant in that we'll cache non-secure resources now, too.

Reply:
Ah, got it!

> Allow insecure external resources to be cached for testing on localhost

Is there a risk here of content being cached in localhost and leading to confusion? I'd almost prefer it kept _not working_ in localhost, but perhaps that's a naive statement?